Resources/Textures/Structures/Machines/anomaly_sync.rsi/meta.json

```diff
@@ -0,0 +1,38 @@
+{
+  "version": 1,
```

Comment:
2 space indent not 1

Reply:
Sitting here comparing my meta to other rsi's metas..... It's a fascinating pastime. To no avail.

Reply:
some use tabs some use 4 spaces some use 1 space the style is 2 space :trollface:

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/96445749/46efaf16-5cdc-478c-bcaa-312a72090e67)
more than half of the meta files use this style
meh 👀

Reply:
yes thats 2 space indent not 1

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/96445749/94b97ec7-1ee0-4227-a368-fb5f604c1d59)
my meta is same!

Reply:
ok you are right grubhub mobile was just on crack :trollface: